org._2006Scape.Launcher.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org._2006Scape.Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.openjdk8
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=~/2006Scape:create
+  - --socket=pulseaudio
+modules:
+- name: openjdk
+  buildsystem: simple
+  build-commands:
+  - "/usr/lib/sdk/openjdk8/install.sh"
+- name: 2006Scape
+  buildsystem: simple
+  build-commands:
+  - install -D run.sh /app/bin/run.sh
+  - install -D 2006Scape_Launcher.jar /app/bin/2006Scape_Launcher.jar
+  - install -D org._2006Scape.Launcher.desktop /app/share/applications/org._2006Scape.Launcher.desktop
+  - install -D icon.svg /app/share/icons/hicolor/scalable/apps/org._2006Scape.Launcher.svg
+  - install -D org._2006Scape.Launcher.appdata.xml /app/share/metainfo/org._2006Scape.Launcher.appdata.xml
+  sources:
+  - type: file
+    path: run.sh
+  - type: file
+    url: https://2006Scape.org/Downloads/2006Scape_Launcher.jar
```

Comment:
it's not versionned? What happen when the next version is out?

Reply:
Does the sha256 not handle this?

Reply:
it will break the build for sure, which is why this should be versioned too.

Reply:
how do you mean it should be versioned? i'll just update the sha256 each time i update the file "https://2006Scape.org/Downloads/2006Scape_Launcher.jar"